src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -335,10 +338,14 @@ function ReportActionItemMessageEdit(props) {
      * @param {String} emoji
      */
     const addEmojiToTextBox = (emoji) => {
-        setSelection((prevSelection) => ({
-            start: prevSelection.start + emoji.length + CONST.SPACE_LENGTH,
-            end: prevSelection.start + emoji.length + CONST.SPACE_LENGTH,
-        }));
+        let newSelection = {
+            start: selection.start + emoji.length + CONST.SPACE_LENGTH,
+            end: selection.start + emoji.length + CONST.SPACE_LENGTH,
+        };
+        setSelection(newSelection);
+
+        if (isMobileChrome) emojiPickerSelectionRef.current = newSelection; // immediately set the selection again on Chrome mobile after focusing the inputm which seems to change the cursor position for a brief moment
```

Comment:
Please use brackets with `if` blocks. It is a common pattern on the project. Also, a minor grammar fix - _input_ instead of _inputm_.
```suggestion
        if (isMobileChrome) {
            // immediately set the selection again on Chrome mobile after focusing the 
            // input which seems to change the cursor position for a brief moment
            emojiPickerSelectionRef.current = newSelection;
        }
```

Reply:
Thanks

Reply:
Addressed in 7fc489c77a22ea300367d5c295ee396de5c24668